config/routes.rb

```diff
@@ -12,4 +12,6 @@
     post 'upvote'
     post 'downvote'
   end
+
+  resources :reviews, only: [:show, :edit, :update, :destroy]
```

Comment:
I don't see a `show` action in your `reviews_controller`, so there shouldn't be a route for it being defined here!